.buildkite/test-amd.yaml

```diff
@@ -187,7 +187,7 @@ steps:
   - tests/distributed/test_utils
   - tests/distributed/test_pynccl
   - tests/distributed/test_events
-  - tests/compile/test_basic_correctness
+  - tests/compile/fullgraph/test_basic_correctness.py
```

Comment:
@gmagogsfm after this PR, t here are some tests in tests/compile/distributed, tests/compile/fullgraph, and also some individual tests under tests/compile (no subdirectory).

When would someone add a test to tests/compile/fullgraph vs tests/compile?

Reply:
I think `tests/compile` should mostly have unit tests, ie tests for a particular code path/feature

tests/compile/fullgraph are e2e full model tests that do not target specific features.

Their distinction mostly derives from previous setup where they had separate time budgets

```
- label: PyTorch Compilation Unit Tests # 15min
- label: PyTorch Fullgraph Smoke Test # 15min
- label: PyTorch Fullgraph Test # 27min
```